### ISSUE TITLE
Review grid layout of navigation block listings

### ISSUE THREAD

**Issue by axelboc** [unimelb/unimelb-design-system#354]
Current layout is based on **inline-blocks**, which adds white space between elements and forces the use of magic numbers in `width` declarations. This, in turn, leads to imprecise alignments of the listings as a whole. (See PR #353.) Investigate switching to a **Flexbox** layout, or switch to using **floats**.

https://web.unimelb.edu.au/components/listings/

**Comment by yevek** [comment]
not sure if we still need this one @axelboc will lump in with rest of listing fixes

**Comment by axelboc** [comment]
Yeah still a must. We'll need to use floats, as flexbox isn't great for things that wrap.

**Comment by axelboc** [comment]
- [ ] default listing
- [ ] feature listing (2 and 3 columns)
- [ ] block listing (news and events)
- [ ] navigation text and block listings
